### PR TITLE
[enhance] Add argument for korean in explore graph paths

### PR DESCRIPTION
**Pull request by Eradan94**
## Description

Add an option "-K" for Fst2List, to detect if Unitex is launched in korean.
This PR refers issue #131 and is associated with PR 88 of unitex-core.
## Motivation and Context

This PR fixes the issue 87 of unitex-core.
## How Has This Been Tested?

See PR 88 in unitex-core.

## Type of files

- [ ] `bin`: Binary files
- [ ] `ci`: Continuous integration files
- [ ] `doc`: Documentation files
- [x] Plain-text source code files

## Level of change

- [ ] `break`: Breaking change
- [ ] `exp`: Experimental change
- [ ] `tmp`: Temporal change
- [ ] `major`: Major change
- [x] `minor`: Minor change
- [ ] `sec`: Vulnerability-related change
- [ ] None of the above (normal change)

## Type of change

- [ ] `deprecat`: Deprecation of a once-stable feature
- [x] `enhance`: Enhancement in existing functionality
- [ ] `fix`: Bug fix
- [ ] `feature`: New feature
- [ ] `hotfix`: Hotfix for bugs
- [ ] `refactor`: Improve coding style, comments
- [ ] `remove`: Remove a feature

## Checklist:

- [x] My code compiles
- [x] My code follows the code style of this project
- [x] My code includes javadoc/doxygen where appropriate
- [x] My code is well factored, so that there is not repetitive code in the wild
- [ ] My code does not require a change in the documentation, if so I already opened an issue to list the changes
- [x] I have read the **CONTRIBUTING** document
- [x] I have read the **Commit Message Guidelines**
- [x] I understand that all commits on my pull request will be squashed to a single good one

### CODE REVIEW

File: unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java [hunk 1]
```diff
@@ -426,6 +427,9 @@ private void runButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIR
         if ( !checkLoopsCheckbox.isSelected() ) {
                 cmd = cmd.noLoopCheck();
         }
+        if (ConfigManager.getManager().isKorean(null)) {
+			cmd = cmd.korean();
```
Comment (martinec) [inline unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java]: Make sure to use the right indentation.

Comment (Eradan94) [reply unitex/src/fr/umlv/unitex/frames/GraphPathFrame.java]: Done.
But I had a problem when I wanted to push the changes, I got this message (I don't know why) : 
! [rejected]          korean -> korean (non-fast-forward)
I resolved my problem but it made 3 commits.

**Comment by martinec** [comment]
LGTM

**Comment by martinec** [comment]
@Eradan94, can you open an issue at gramlab-ide for this PR? Thank you.

**Comment by Eradan94** [comment]
> @Eradan94, can you open an issue at gramlab-ide for this PR? Thank you.

Done

**Pull request merged**